README.md

```diff
@@ -256,28 +256,38 @@ Note: This requirement and the requirement to only use secure protocols makes pa
 
 ## Detailed Design
 
-### Opt-in cookie attribute
-
-We propose a new cookie attribute, `Partitioned`, which must be specified by the `Set-Cookie` header to indicate that the cookie should be stored in a per-top-level partition.
-Any cookies that are not set with the `Partitioned` attribute will eventually be blocked in third-party contexts.
-(Note: Other features like the [`SameParty`](https://github.com/cfredric/sameparty) attribute may adjust the details of such blocking across domains within the same First-Party Set.)
+### Partitioning model
 
-### Partition by top-level context
+Today, cookies are keyed on the hostname or domain of the site that set them, i.e. their _host key_.
+After CHIPS, cookies that opt into partitioning will be double-keyed on their _partition key_ and their _host key_.
+A cookie's partition key is the [site](https://html.spec.whatwg.org/#sites) (i.e. scheme and registrable domain) of the top-level URL the browser was visiting at the start of the request to the endpoint that set the cookie.
 
-Cookies set with the `Partitioned` attribute must only be sent to the third-party host when the user agent is in the same _top-level context_ as when the cookie was first set.
+Likewise, a request's partition key is the site of the top-level URL the browser is visiting at the start of a request.
+Browsers must only send a cookie with the `Partitioned` attribute in requests with the same partition key as that cookie.
 
-For most sites, the top-level context is just the top-level site. However, when the top-level site is part of a [First-Party Set](https://github.com/privacycg/first-party-sets), the third-party can share the same partition across sites within the same set.
+If the top-level site has an `https` scheme and is part of a [First-Party Set](https://github.com/privacycg/first-party-sets), the third-party can share the same partition across sites within the same set.
+In this case, the domain component of the partition key is the [owner domain](https://github.com/privacycg/first-party-sets#declaring-a-first-party-set) of the First-Party-Set.
 This is consistent with Chrome’s [privacy principle of partitioning identity by first party](https://github.com/michaelkleber/privacy-model#identity-is-partitioned-by-first-party-site), and ensures that tracking across unrelated sites is prevented by the obsoletion of unpartitioned third-party cookies.
 
+If the top-level site has an `http` scheme, the First-Party Set feature [does not apply](https://github.com/privacycg/first-party-sets#declaring-a-first-party-set), since it is only support for origins with an https scheme.
+
+If the top-level site has an `http` scheme, the First-Party Set feature does not apply, since all sets must contain only secure origins.
```

Comment:
This looks like a duplicate of the statement above?

Reply:
Removed.